zuul.d/jobs.yaml

```diff
@@ -152,6 +152,7 @@
     parent: upload-docker-image
     vars:
       docker_registry: quay.io
+    nodeset: ubuntu-focal
```

Comment:
well, actually instead please change nodeset of the otcinfra-upload-image and otcinfra-promote-image.

Reply:
done